README.md

```diff
@@ -1,21 +1,28 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
-
-(b) Where in your JS file should you put it, and WHY? 
-
+(a) What is a click handler?
+  A click handler tells the browser what to do, when there is a click on a button, and where to retrieve the data that is being requested.
+(b) Where in your JS file should you put it, and WHY?
+  With Tom's articles example in the repo from the other day, it is in the article_list.js file. This is because the click is the function performed to show the list of articles. You can not see/access the list of articles is there is no click function.
```

Comment:
A click handler should go in the `$(document).ready()`, assuming you want it to be available immediately on page load.